router/batchrouter/asyncdestinationmanager/bing-ads/bingads.go

```diff
@@ -55,94 +55,152 @@ func generateClientID(user User, metadata Metadata) string {
 	return strconv.FormatInt(jobId, 10) + "<<>>" + user.HashedEmail
 }
 
-func (b *BingAdsBulkUploader) CreateZipFile(filePath, audienceId string) (string, []int64, []int64, error) {
-	failedJobIds := []int64{}
-	successJobIds := []int64{}
+func csvZipFileCreator(audienceId string, fileType string) (string, *csv.Writer, string) {
```

Comment:
fileType is same as action type? then please use that as name

Reply:
@ItsSudip